Signal/src/ViewControllers/MediaPageViewController.swift

```diff
@@ -0,0 +1,706 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import UIKit
+
+class MediaPageViewController: UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate, MediaDetailViewControllerDelegate {
+
+    private struct MediaGalleryItem: Equatable {
```

Comment:
can you please log dealloc as we do in `OWSViewController`?  It's pretty easy to introduce leaks in the views.

Reply:
good call, done.